packages/@tinacms/cli/src/cmds/index.ts

```diff
@@ -0,0 +1,17 @@
+interface CommandHandler<T, U> {
```

Comment:
Might be nice to add a short comment of what T and U are or rename them to something more declarative.